src/components/LogoutInfo.tsx

```diff
@@ -8,10 +8,12 @@ const LogoutInfoComponent = (): React.ReactElement => {
   const user = getUser();
   const name = user ? `${user.given_name} ${user.family_name}` : '';
   return (
-    <div className={styles['content-element']}>
-      <h3>Olet kirjautunut, {name}</h3>
+    <div className={styles['content-element']} data-test-id="logout-info">
+      <h3>
+        Olet kirjautunut, <span data-test-id="logout-user-name">{name}</span>
+      </h3>
       <div>
-        <Button translate="" onClick={logout}>
+        <Button translate="" onClick={logout} data-test-id="logout-info">
```

Comment:
There's the same test id, `logout-info`, in two elements. Isn't that a bad thing?

Reply:
Logout button is not used in tests, so I removed the test-id from it.